src/settings.js

```diff
@@ -2060,6 +2060,11 @@ var RUNTIME_DEBUG = false;
 // library symbol.
 var LEGACY_RUNTIME = false;
 
+// User-defined functions to wrap with signature conversion
```

Comment:
This comment should mention that this is for documenting native functions that take or return pointer arguments and that it only effects wasm64.